packages/app/src/preloadApi.ts

```diff
@@ -61,4 +61,20 @@ export const modulesApi = {
     isUpdateAvailable: (...args: any[]) => ipcRenderer.invoke('wowarenalogs:app:isUpdateAvailable', ...args),
     clearStorage: (...args: any[]) => ipcRenderer.invoke('wowarenalogs:app:clearStorage', ...args),
   },
+  obs: {
+    selectFolder: (...args: any[]) => ipcRenderer.invoke('wowarenalogs:obs:selectFolder', ...args),
+    startRecording: (...args: any[]) => ipcRenderer.invoke('wowarenalogs:obs:startRecording', ...args),
+    stopRecording: (...args: any[]) => ipcRenderer.invoke('wowarenalogs:obs:stopRecording', ...args),
+    getConfiguration: (...args: any[]) => ipcRenderer.invoke('wowarenalogs:obs:getConfiguration', ...args),
```

Comment:
a little weird that a module has its own configuration. can this be done within the app level config?

Reply:
In theory yes but in practice it's much more complex than you might imagine -- the manager class itself handles diffing its own config and managing when to re-start the OBS buffering process (some of the args require an OBS restart, some do not). The restart is somewhat async as well so there's some promise handling to make sure the updates all go through.

Ultimately we *could* expose the individual config settings out as setter/getter methods from the module but I think the value add for us of doing this needs to be pretty high to justify all the extra cruft.